src/components/data-entry/QueryItem/NumberInput.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { type Meta, type StoryObj } from '@storybook/react'
+import { QueryItem } from 'src/components'
+
+const meta: Meta<typeof QueryItem.ValueSelector.NumberInput> = {
+  title: 'Aquarium/Data Entry/QueryItem/ValueSelector/NumberInput',
+  component: QueryItem.ValueSelector.NumberInput,
+  parameters: {
+    docs: {
+      description: {
+        component: 'This is the "Number Input" component of the QueryItem/ValueSelector component group.',
+      },
+    },
+  },
+  args: {
+    onChange: (value: string | number) => {
```

Comment:
Can't be a string since we are calling `parseFloat`
```suggestion
    onChange: (value: number) => {
```